README.md

```diff
@@ -30,15 +30,27 @@ const checkoutInstance = startCheckout(manifest, [options])
+| `id`          | `string`   | no       | `'easypay-checkout'` | The id of the HTML element where the Checkout form should be included.     |
+| `onSuccess`   | `function` | no       | `() => {}`           | Callback function to be called when the Checkout is finished successfully. |
+| `onError`     | `function` | no       | `() => {}`           | Callback function to be called on errors.                                  |
+| `onClose`     | `function` | no       | `undefined`          | Callback function to be called when the Checkout interaction is closed.    |
+| `testing`     | `boolean`  | no       | `false`              | Whether to use the testing API (`true`) or the production one (`false`).   |
+| `display`(1)  | `string`   | no       | `'inline'`           | The display style of the element that hosts the Checkout.                  |
+| `hideDetails` | `boolean`  | no       | `false`              | Whether to hide the details form or not. An expandable summary will be shown with the details, instead. |
+| `language`(2) | `string`   | no       | `undefined`          | The language in which to display the Checkout.                             |
+| `logoUrl`               | `string`  | no       | `undefined`     | The merchant logo url to display in the Checkout.                           |
+| `backgroundColor`       | `string`  | no       | `'#ffffff'`     | The color used as the background of the Checkout page.                      |
+| `accentColor`           | `string`  | no       | `'#0d71f9'`     | The color used in highlights, as well as default buttons and input borders. |
+| `errorColor`            | `string`  | no       | `'#ff151f'`     | The color used for errors.                                                  |
+| `inputBackgroundColor`  | `string`  | no       | `'transparent'` | The color used for the input backgrounds.                                   |
+| `inputBorderColor`      | `string`  | no       | *accentColor*   | The color for input borders.                                                |
+| `inputBorderRadius`     | `number`  | no       | `50`            | The border radius for inputs, in `px`.                                      |
+| `inputFloatingLabel`    | `boolean` | no       | `true`          | Whether inputs should use floating labels.                                  |
+| `buttonBackgroundColor` | `string`  | no       | *accentColor*   | The color used for the button backgrounds.                                  |
+| `buttonBorderRadius`    | `number`  | no       | `50`            | The border radius for buttons, in `px`.                                     |
+| `buttonBoxShadow`       | `boolean` | no       | `true`          | Whether the buttons should have box-shadow.                                 |
+| `fontFamily`            | `string`  | no       | `'Overpass'`    | The font used in the text.                                                  |
```

Comment:
instead of `The font used in the text` I think `The font used for the text` suits better no?